lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1545,6 +1552,15 @@ LogicalResult ModuleEmitter::visitSV(InterfaceModportOp op) {
   return success();
 }
 
+LogicalResult ModuleEmitter::visitSV(AssignInterfaceSignalOp op) {
+  SmallPtrSet<Operation *, 8> emitted;
+  indent() << "assign ";
```

Comment:
Did you consider a projection of the modport as an `rtl.inout` value?  This would allow you to use the existing `ConnectOp`.  This would also allow working with the SV procedural assignment forms.